components/InputForm/index.js

```diff
@@ -0,0 +1,94 @@
+import StyledButton from "../button";
+import StyledInput from "./Input";
+import StyledGrid from "./Grid";
+import StyledForm from "./Form";
+import React from "react";
```

Comment:
That can go...

Reply:
I need it because of the `React.Fragment` I use. I need the `React.Fragment` to use the `key`prop